borsh/src/ser/helpers.rs

```diff
@@ -0,0 +1,23 @@
+use crate::maybestd::{
+    io::{Result, Write},
+    vec::Vec,
+};
+use crate::BorshSerialize;
+
+/// Serialize an object into a vector of bytes.
+#[inline]
```

Comment:
Curious, I'd say that this inline isn't necessary, as the function is generic anyway. OTOH, serde_json has inline here, but it's hard to find rational for that, it was some pre-existing code: https://github.com/serde-rs/json/commit/c79213f8a200b1abdbc83c56387bacc5a2418ce2#diff-a00e624afb2c1ca37bc85c5bf4fd7183292984533b47d5ce940cb77bf35ded55R586. So, :shrug:, it doesn't matter much either way.

Reply:
good point, removed